Description: Fixed html template location
Author: David GUENAULT <dguenault@monitoring-fr.org>
Origin: upstream, https://github.com/nicolargo/glances/commit/94d016ab517a8ba5dd47a1bff7105d8ba7a8a4cc
Bug: https://github.com/nicolargo/glances/issues/122
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/glances/glances.py
+++ b/glances/glances.py
@@ -2085,7 +2085,7 @@
 
         # Set the templates path
         environment = jinja2.Environment(
-            loader=jinja2.FileSystemLoader(self.root_path + 'html'),
+            loader=jinja2.FileSystemLoader(os.path.dirname(__file__) + '/html'),
             extensions=['jinja2.ext.loopcontrols'])
 
         # Open the template
